tools/cldr-apps/js/src/esm/cldrVettingParticipation.js

```diff
@@ -115,6 +117,8 @@ async function downloadVettingParticipation(opts) {
       "CldrCovCount",
       "ProgressVote",
       "ProgressCount",
+      "ProgressPercent",
```

Comment:
Minor: for spreadsheets it is better to have spaces in titles so that they wordwrap. So

"Progress Vote",
"Progress Count",
"Progress Percent",

Also, we should have somewhere a key to what these columns are. Can be a Sites page. For example, I don't know off-hand what "Votes" means or what "CldrCovCount" means. I'm guessing:

Votes = total number of path values in this locale that have a vote from this vetter, possibly including paths that are above the organization's coverage target for the locale (such as comprehensive)

CldrCovCount = total number of paths that are in your organization's coverage target for this locale

Reply:
Ok and I can add a spreadsheet comment also with more info 

Thanks for the feedback on the wording. 

The percent column isn't working yet but otherwise this is close.

Reply:
@macchiati I think  it's:

> CldrCovCount = total number of paths that are in *CLDR's* coverage target for this locale

hence the CLDR in the name.

Reply:
done